scanner/src/main/kotlin/ScanStorages.kt

```diff
@@ -19,6 +19,8 @@
 
 package org.ossreviewtoolkit.scanner
 
+import org.ossreviewtoolkit.model.Identifier
```

Comment:
Again, a teaser for the planned context in which the function will be used would be nice.